subprojects/test-runner/test-inhouse-runner/src/main/kotlin/com/avito/android/rule/InHouseScenarioScreenRule.kt

```diff
@@ -16,13 +16,16 @@ import org.junit.rules.TestRule
 import org.junit.runner.Description
 import org.junit.runners.model.Statement
 
-abstract class InHouseScenarioScreenRule<A : Activity>(activityClass: Class<A>) : TestRule {
+abstract class InHouseScenarioScreenRule<A : Activity>(
+    activityClass: Class<A>,
+    stubIntents: Boolean = true
```

Comment:
Current behaviour

Reply:
Will we make it disabled by default in the next release? What are your plans?

Reply:
I'm asking because this new flag will help to disable stubs but won't help to diagnose the problem.
The question is about making a less error-prone API.

Reply:
I was afraid if I turned it off by default, I'd break the tests. I'll check the integration against Avito

Reply:
> if I turned it off by default, I'd break the tests

Most probably, but this is not a blocker to change behavior. It can be done in multiple steps.
The question is not about changes in this PR but rather about the final goal.

Reply:
Integration build succeed. Changed the flag value to `true`

Reply:
I was wrong, the build failed.